src/pages/ChosenTopicPage.jsx

```diff
@@ -0,0 +1,10 @@
+import ArticleList from "../component/ArticleList"
```

Comment:
This component doesn't really appear to be doing a whole lot that isn't taken care of elsewhere and is adding to the size of your app unnecessarily.